Added common tracking cookies #80401

AdamWr opened the pull request:
https://github.com/AdguardTeam/AdguardFilters/issues/80401

AdamWr committed:
Added common tracking cookies #80401

wout-codes commented:
Maybe also add: `$cookie=adEdition` from the following site: `https://www.nbc.com/`.

AdamWr committed:
🍪

Added cookies from:
mparticle.com
Adobe
nbcuniversal.com
ads.stickyadstv.com
gemius.com
Taboola

AdamWr committed:
More 🍪

Added:
Hotjar
Microsoft/Bing
LinkedIn
Smartadserver
Eulerian
Pubmatic
bidswitch.net
Amazon
Sitecore

AdamWr committed:
Update cookies.txt

Added:
Optimizely
session_tracker - seems to be reddit specific

AdamWr committed:
Update cookies.txt

AdamWr committed:
Update cookies.txt

Added:
Outbrain
Cxense
adtlgc.com
yTrack

ameshkov commented on SpywareFilter/sections/cookies.txt:
```diff
@@ -34,4 +33,142 @@
-$cookie=utag_main
\ No newline at end of file
+$cookie=utag_main
+! Google - https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage
```
it would be really cool if we could add a comment to every cookie (where this is possible).

The thing is that those pages you're linking are changing with time and we'll simply lose this info if we don't record it here.

Alex-302 replied:
Some cookies with description listed in tables:
https://www.staqq.co/cookie-policy
https://support.devex.com/hc/en-us/articles/115000959453-Devex-Privacy-Statement

AdamWr replied:
Added Reblaze and tinypass cookies and some comments.

ameshkov commented on SpywareFilter/sections/cookies.txt:
```diff
@@ -34,4 +33,142 @@
+$cookie=/^CaosGtag_ga/
+$cookie=_gaexp
+$cookie=_gaexp_rc
+$cookie=utma
```
Should be `__utma` (other `utm*` cookies as well)

AdamWr replied:
Right, I will change it.
By the way, this website `https://www.qarson.fr/` is using `utma`, so maybe we should add both rules, a least for `utma`.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/114750837-b7b77780-9d54-11eb-9147-bbbe87520742.png)

</details>

AdamWr replied:
Already there is `__utma` and some other rules which starts with `__` below - https://github.com/AdguardTeam/AdguardFilters/pull/80418/files#diff-502809f1020ded4a26ff4ad467ffe53d0e3f42db352794e3537d2910604ea95bR64

ameshkov replied:
> Right, I will change it.
> By the way, this website https://www.qarson.fr/ is using utma, so maybe we should add both rules, a least for utma.

Seems to be a custom cookie they're setting.

AdamWr committed:
fixed Google cookies

AdamWr committed:
Added comments

AdamWr committed:
Update cookies.txt

Added:
Reblaze
tinypass.com

AdamWr committed:
Merge branch 'master' into tracking_cookies

AdamWr committed:
Merge branch 'master' into tracking_cookies
